addons/pos_sale/data/pos_sale_data.xml

```diff
@@ -6,8 +6,8 @@
         </record>
 
         <record id="pos_sale.default_downpayment_product" model="product.product">
-            <field name="name">Down Payment (POS)</field>
```

Comment:
If we remove this, it won't be clear it's only used in the pos.
Furthermore, it's not a demo data, it's a data, the spec is wrong/confusing.

Reply:
@thomasdeleval Your opinion?